src/main/java/com/zendesk/maxwell/metrics/MaxwellHealthCheck.java

```diff
@@ -0,0 +1,19 @@
+package com.zendesk.maxwell.metrics;
+
+import com.codahale.metrics.health.HealthCheck;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class MaxwellHealthCheck extends HealthCheck {
+
+	static final Logger LOGGER = LoggerFactory.getLogger(MaxwellHealthCheck.class);
+
+	public MaxwellHealthCheck() { }
+
+	@Override
+	protected Result check() throws Exception {
+		// TODO: actually return something useful
+		return Result.healthy();
```

Comment:
it assumes something about the system though.
in our case, we're about to have new alpha-version services running production. these will get very sporadic use for the first few months as we introduce select customers to them and get feedback. i feel like that scenario would trip most (all?) non-configurable thresholds.